README.md

```diff
@@ -78,3 +78,10 @@ Rspec/Focused:
 3. Commit your changes (`git commit -am 'Add some feature'`)
 4. Push to the branch (`git push origin my-new-feature`)
 5. Create a new Pull Request
+
+
+For running the spec files, this project depends on RuboCop's spec helpers. This means that in order to run the specs locally, you need a (shallow) clone of the RuboCop repository:
+
+```bash
+git submodule update --init --depth 1 vendor/rubocop
```

Comment:
Apparently this is a known problem.

> > As far as I can tell this option isn't usable for submodules which don't track `master` very closely. If you set depth 1, then `submodule update` can only ever succeed if the submodule commit you want is the latest master.  **Otherwise you get "`fatal: reference is not a tree`"**.

> That is true.
> That is, until git 2.8 (March 2016). With 2.8, the `submodule update --depth` has one more chance to succeed, even if the SHA1 is directly reachable from one of the remote repo HEADs.

> http://stackoverflow.com/a/17692710/4353

I'll just update the instruction in README.